dags/resources/stages/parse/table_definitions/convex/ConvexFraxPool_event_LockedLonger.json

```diff
@@ -0,0 +1,76 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": true,
+                    "internalType": "address",
+                    "name": "user",
+                    "type": "address"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "bytes32",
+                    "name": "kek_id",
+                    "type": "bytes32"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "new_secs",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "new_start_ts",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "new_end_ts",
+                    "type": "uint256"
+                }
+            ],
+            "name": "LockedLonger",
+            "type": "event"
+        },
+        "contract_address": "(SELECT DISTINCT _stakingAddress FROM ref(`FraxBooster_call_addPool`) UNION ALL SELECT * FROM UNNEST(['0x02577b426f223a6b4f2351315a19ecd6f357d65c', '0x0a08673e3d7c454e1c6b27acd059c50df6727fc9', '0x698137c473bc1f0ea9b85ade45caf64ef2df48d6']))",
```

Comment:
Please use a set of UNION ALL instead of UNNEST from an array like in this example https://github.com/blockchain-etl/ethereum-etl-airflow/blob/master/dags/resources/stages/parse/table_definitions/curve/CryptoSwap_event_RemoveLiquidityOne.json#L30. There is a limitation in BigQuery that prevents us using UNNEST in a subquery